features/step_definitions/login_steps.rb

```diff
@@ -0,0 +1,46 @@
+Given(/^I am on login page$/) do
+  visit LoginPage
+end
+
+When(/^I click at arrow button$/) do
+  on(LoginPage) do |login_page|
+    login_page.access_to_another_acc_element.click if login_page.access_to_another_acc_element.exists?
```

Comment:
why you are using if? "I click at arrow button" means that button exist. Review other steps